common/culture/cultures/wc_aqir.txt

```diff
@@ -41,7 +41,7 @@ nerubian = {
 		tradition_stalwart_defenders
 		tradition_philosopher_culture
 		tradition_city_keepers
-		tradition_strength_in_numbers 
+		tradition_strength_in_numbers
```

Comment:
i'm wondering what's your criteria for choosing which cultures have beacon of learning versus which have philosopher culture?

Reply:
Well, as discussed previously with @ElMariuso, it'd be the cultures that are not completely shut off from the rest of the world.
Which is why nerubians and dragons didn't get it.
Other cultures like High Elves and Shadowmoon are famed for being centers of learning. The other orc clans tended to look at the Shadowmoon for guidance. And many people interacted the elves.

Reply:
Although they could still be relatively isolated and still pride themselves as a center of learning, even if it's amongst themselves.